core/src/main/java/com/opensymphony/xwork2/config/providers/XmlConfigurationProvider.java

```diff
@@ -777,11 +777,21 @@ protected Class verifyResultType(String className, Location loc) {
                 }
                 params.putAll(resultParams);
 
-                ResultConfig resultConfig = new ResultConfig.Builder(resultName, resultClass)
-                        .addParams(params)
-                        .location(DomHelper.getLocationObject(element))
-                        .build();
-                results.put(resultConfig.getName(), resultConfig);
+                Set<String> resultNamesSet;
+                if (",".equals(resultName.trim())) {
+                    resultNamesSet = new HashSet<>(1);
+                    resultNamesSet.add(resultName);
```

Comment:
Currently it is allowed to have a result name as a `,`. And `TextParseUtil.commaDelimitedStringToSet` will produce empty set in that case. Note that it is trimmed so the result name can be something like "&nbsp;&nbsp;&nbsp;," also.